components/chart/trendlines.md

```diff
@@ -0,0 +1,320 @@
+            };
+
+            return data;
+        }
+
+        public static List<SalesData> GenerateScatterData()
+        {
+            var data = SalesData.GenerateData();
+
+            for (int i = 0; i < data.Count; i++)
+            {
+                data[i].NumericPeriod = i + 10;
+            }
+
+            return data;
+        }
+    }
+}
+`````
+
+### Moving Average Trendline
+
+The Moving Average Trendline smoothens out data fluctuations by computing an average of all data points over a specified period. By default, this period is set to `2` chart intervals.
+
+>caption The following example showcases a moving average trendline in line series:
+
+`````CSHTML
+<TelerikChart Width="@Width" Height="@Height">
+    <ChartPannable Enabled="true" Lock="@ChartAxisLock.Y"></ChartPannable>
+    <ChartZoomable Enabled="true"></ChartZoomable>
+    <ChartLegend Visible="true" Position="@ChartLegendPosition.Bottom"></ChartLegend>
+    <ChartTooltip Visible="true" Shared="true"></ChartTooltip>
+
+    <ChartSeriesItems>
+        <ChartSeries Type="ChartSeriesType.Line"
+                     Name="@Series1Name"
+                     Data="@Data"
+                     Field="@nameof(SalesData.Count)"
+                     CategoryField="@nameof(SalesData.Period)">
+            <ChartSeriesMarkers Visible="false"></ChartSeriesMarkers>
+        </ChartSeries>
+
+        <ChartSeries Type="ChartSeriesType.MovingAverageTrendline"
+                     Name="Sales Forecast (MOVING AVERAGE)"
+                     For="@Series1Name"
+                     Color="teal">
+            <ChartSeriesMarkers Visible="false"></ChartSeriesMarkers>
+            <ChartSeriesTrendline>
+                <ChartSeriesTrendline Period="4"></ChartSeriesTrendline>
+            </ChartSeriesTrendline>
+        </ChartSeries>
+    </ChartSeriesItems>
+
+    <ChartCategoryAxes>
+        <ChartCategoryAxis>
+            <ChartCategoryAxisLabels Position="@ChartAxisLabelsPosition.Start">
+                <ChartCategoryAxisLabelsRotation Angle="-90"></ChartCategoryAxisLabelsRotation>
+            </ChartCategoryAxisLabels>
+        </ChartCategoryAxis>
+    </ChartCategoryAxes>
+
+    <ChartValueAxes>
+        <ChartValueAxis Min="-4e5" Max="1.2e6"></ChartValueAxis>
+    </ChartValueAxes>
+</TelerikChart>
+
+@code {
+    public const string Height = "400px";
+    public const string Width = "100%";
+    public const string Series1Name = "Sales By Quarter";
+
+    List<SalesData> Data { get; set; } = new List<SalesData>();
+
+    protected override Task OnInitializedAsync()
+    {
+        Data = SalesData.GenerateData();
+        return base.OnInitializedAsync();
+    }
+
+    public class SalesData
+    {
+        public string Period { get; set; }
+        public int NumericPeriod { get; set; }
+        public DateTime Date { get; set; }
+        public int Count { get; set; }
+
+        public static List<SalesData> GenerateData()
```

Comment:
It's strange architecture to hard-code data in a the model class. Move this to a separate method and call it from `OnInitializedAsync`.